content/article/cardano-cube-find-every-project-thats-building-on-the-cardano-blockchain.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Cardano Cube - Find every project that's building on the Cardano Blockchain
+tags:
+  - platform
+  - Ecosystem
+  - dAppStore
+url: ""
+image: https://ucarecdn.com/8a643b14-d54f-486f-90ed-bef4dcde2618/
+image_text: Cardano Ecosystem Map
+---
+
+It’s no secret that cryptocurrency Cardano is growing every day. With a massive amount of ADA staked by it’s investors, it looks like a promising cryptocurrency for the long-term. Many people in the cryptocurrency community even label the project as “The blockchain of the future.”
+
+As there are many projects and dApps launching on the Cardano platform, there’s also a lot of investors and community members looking for the ‘next big thing’. At [Cardano Cube](https://www.cardanocube.io/), we are known for informing investors about projects building on the Cardano blockchain. Our platform containing hours of precious research about almost all of the Cardano projects.
+
+## **An interactive Cardano Ecosystem Map**
+
+We’ve recently released our [Interactive Cardano Ecosystem Map](https://www.cardanocube.io/cardano-ecosystem-interactive-map). A beautiful interactive map where all projects that are building on Cardano are being listed and shown in their respective industry. Simple, easily scannable, fully responsive and most of all: “updated every day”.
+
+The Cardano community uses the interactive map to explore projects and read about their benefits, price, team and background. In total the Cardano Cube platform contains over 1000 hours of dedicated Cardano research.
+
+## **About Cardano Cube**
```

Comment:
The H2 heading doesn't need to be formatted in bold. Please delete '**' from the heading